File: src/main/java/me/andre111/d20server/message/util/Ping.java
package me.andre111.d20server.message.util;

import me.andre111.d20server.message.Message;
import me.andre111.d20server.message.RecievableMessage;
import me.andre111.d20server.message.UnauthenticatedMessage;

public class Ping extends RecievableMessage implements UnauthenticatedMessage {
	public long time;
	
	@Override
	public Message handle() {
		return this;
	}
}

File: core/common/message/util/ping.js
// @ts-check
import { Message } from '../message.js';
import { registerType } from '../../util/datautil.js';

export class Ping extends Message {
    time;

    constructor(time) {
        super();
        this.time = time;
    }

    getTime() {
        return this.time;
    }
}
registerType(Ping);
